cl/api/templates/rest-docs-vlatest.html

```diff
@@ -1103,5 +1088,22 @@ <h2 id="copyright"><span>Copyright</span></h2>
       </a>
     </p>
 
+    <div class="well v-offset-above-3">
+      <h2 class="top text-center">Please Support Open Legal Data</h2>
+      <p>These bulk data files are sponsored by <a href="https://free.law">Free Law Project</a> and users like you. We provide these files in furtherance of our mission to make the legal sector more innovative and equitable.
+      </p>
+      <p>We have provided these files for over a decade, and we need your contributions to continue curating and enhancing this service.
+      </p>
+      <p>Will you support us today with a donation?</p>
+      <div class="row">
+        <div class="hidden-xs col-sm-1 col-md-2 col-lg-3"></div>
+        <div class="col-xs-12 col-sm-10 col-md-8 col-lg-6">
+          <p class="text-center">
+            <a href="{% url "donate" %}?referrer={{ request.path }}" class="btn btn-lg btn-block btn-danger"><i class="fa fa-heart-o"></i> Donate Now</a>
+          </p>
+        </div>
+      </div>
+    </div>
```

Comment:
We should avoid repeating ourselves. Mind putting this into an include and then tweaking this and the other page where it appears?

Reply:
Done.